src/components/App.jsx

```diff
@@ -7,10 +7,10 @@ import classNames from 'classnames'
 import Sidebar from './Sidebar'
 import Alerter from './Alerter'
 
-const App = ({ t, children }) => (
+export const App = ({ t, children }) => (
   <div className={classNames(styles['pho-wrapper'], styles['coz-sticky'])}>
 
-    <Sidebar />
+    <Sidebar t={t} />
```

Comment:
Well, the general consensus in the react community is that context helpers like I18n or routing ones should be composed into components with HOCs, and I think that's better for readability and maintenability. But you're right, @CPatchane may have done this in order to simplify testing (if that's the case, a simple pattern worth exploring would be to made a secondary export of the unwrapped component).